korgan/claim_exemplar_architecture.py

```
@@ -0,0 +1,176 @@
+    if "филиал" not in context and "представительств" not in context:
+        if any("филиал" in x.lower() or "представительств" in x.lower() for x in draft.legal_basis):
+            issues.append("Норма о филиале/представительстве не имеет фактического якоря и должна быть удалена.")
+
+    for line in draft.legal_basis:
+        low = line.lower()
+        mentioned = set(re.findall(r"(?:стать(?:я|и|ей)|ст\.)\s*(\d{1,4})", low))
+        base = re.search(r"основан\w*\s*[:\-]\s*([^\]\n]+)", low)
+        if mentioned and base:
+            grounded = set(re.findall(r"\b(\d{1,4})\b", base.group(1)))
+            if grounded and mentioned.isdisjoint(grounded):
+                issues.append("Несогласованная цитата: номер статьи в тезисе не совпадает с указанным основанием.")
```

Ruff (0.16.2)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)